### PR TITLE
Sn map put army

### PR DESCRIPTION
**Pull request by Kert**

### CODE REVIEW

File: ironfist/src/cpp/adventure/adv.cpp [hunk 1]
```diff
@@ -137,4 +137,16 @@ void advManager::DoEvent(class mapCell *cell, int locX, int locY) {
 		ScriptSignal(SCRIPT_EVT_VISIT_CAMPFIRE, tmp.str());
 	}
 	this->DoEvent_orig(cell, locX, locY);
+}
+
+int advManager::MapPutArmy(int x, int y, int monIdx, int monQty) {
+	int cellIdx = x * gpGame->map.height + y;
+	gpGame->map.tiles[cellIdx].objectIndex = monIdx;
+	gpGame->map.tiles[cellIdx].extraInfo = monQty;
+	gpGame->map.tiles[cellIdx].objTileset = 12;
```
Comment (jkoppel) [inline ironfist/src/cpp/adventure/adv.cpp]: We should already have constants for this. If they're not already defined in some .h file, then you can get them from  HEROES2W.h

Comment (Kert) [inline ironfist/src/cpp/adventure/adv.cpp]: I tried to find something related this but no success.

Comment (jkoppel) [inline ironfist/src/cpp/adventure/adv.cpp]: Oh. Looks like this is actually in EDITOR2W.h . This one is OBJ_TILESET. Go ahead and put that enum somewhere.

File: ironfist/src/cpp/adventure/map.h [hunk 2]
```diff
@@ -377,6 +377,61 @@ enum ADVENTURE_MAP_LOCATION {
   TILE_HAS_EVENT = 128,
 };
 
+enum OBJ_TILESET : __int8 {
```
Comment (jkoppel) [inline ironfist/src/cpp/adventure/map.h]: :D

**Pull request merged**